defaults/main.yml

```diff
@@ -1,9 +1,18 @@
-dynamodb_port: 18000
-dynamodb_version: 2015-04-27_1.0
-dynamodb_sha: 16c074a787af2572e8d6867c2cff47c02fe1b2eb6c956ae2ca604d844d95331c
+dynamodb_port: 8000
+dynamodb_version: 2016-05-17
+dynamodb_sha: 832d474316c9d092de8c24885e69c9be9e22b9807e7383e0de6f247e44601a90.
 dynamodb_source_url: "https://s3-us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_{{ dynamodb_version }}.tar.gz"
+# directories
 dynamodb_install_dir: "/opt/dynamodb"
-dynamodb_log_file: "/opt/dynamodb/dynamodb.log"
+dynamodb_log_dir: "/var/log/dynamodb"
+dynamodb_data_dir: "/var/lib/dynamodb"
+# data files
 dynamodb_log_rotate_file: "/etc/logrotate.d/dynamodb"
+dynamodb_log_file: "{{ dynamodb_log_dir }}/dynamodb.log"
 dynamodb_pid_file: "/var/run/dynamodb.pid"
+# run scripts
 dynamodb_init_script: "/etc/init.d/dynamodb"
+dynamodb_service_file: "/lib/systemd/system/dynamodb.service"
```

Comment:
`/lib/systemd` should be reserved to unit files provided by vendor.
`/etc/systemd` should be preferred.